core/src/main/java/bisq/core/account/sign/SignedWitnessService.java

```diff
@@ -295,4 +313,45 @@ private boolean verifyDate(SignedWitness signedWitness, long childSignedWitnessD
     void addToMap(SignedWitness signedWitness) {
         signedWitnessMap.putIfAbsent(signedWitness.getHashAsByteArray(), signedWitness);
     }
+
+    private void publishSignedWitness(SignedWitness signedWitness) {
+        if (!signedWitnessMap.containsKey(signedWitness.getHashAsByteArray())) {
+            p2PService.addPersistableNetworkPayload(signedWitness, false);
+        }
+    }
+
+    // Arbitrator signing
+    public List<BuyerDataItem> getBuyerPaymentAccounts(long safeDate) {
+        return disputeManager.getDisputesAsObservableList().stream()
+                .filter(this::hasChargebackRisk)
+                .filter(this::isBuyerWinner)
+                .map(this::getBuyerData)
+                .filter(Objects::nonNull)
+                .filter(buyerDataItem -> buyerDataItem.getAccountAgeWitness().getDate() < safeDate)
+                .distinct()
+                .collect(Collectors.toList());
+    }
+
+    private boolean hasChargebackRisk(Dispute dispute) {
+        return PaymentMethod.hasChargebackRisk(dispute.getContract().getPaymentMethodId());
```

Comment:
we should stop calling static util methods. Maybe we can move some methods from PaymentMethod to a class that we can inject.